### ISSUE TITLE
False positive: Add #include <algorithm> for sort

### ISSUE THREAD

**Issue by rakhimov** [google/styleguide#174]
Please see https://github.com/theandrewdavis/cpplint/issues/27

**Comment by mortenfyhn** [comment]
I have another very similar false positive:

Using a minimal example file `test.h` with contents

```cpp
// Copyright 2019 Morten Fyhn Amundsen

#ifndef TEST_H_
#define TEST_H_

void copy(int) {}

#endif  // TEST_H_
```

I also get a false positive suggesting to include the algorithm header:

```
$ cpplint/cpplint.py test.h
test.h:6:  Add #include <algorithm> for copy  [build/include_what_you_use] [4]
Done processing test.h
Total errors found: 1

```

I'm on the current version of the `gh-pages` branch (9c6b371).

---

More information:

```cpp
void copy() {}
```

gives no errors (no missing algorithm header message). However,

```cpp
void copy(void) {}
```

causes cpplint to report the missing algorithm header error.

**Comment by vapier** [comment]
public cpplint development in this repository has been halted (see https://github.com/google/styleguide/pull/837). please migrate to https://github.com/cpplint/cpplint now.
